pkg/api/mlflow/dao/repositories/metric.go

```diff
@@ -26,10 +26,15 @@ type MetricRepositoryProvider interface {
 	// GetMetricHistories returns metric histories by request parameters.
 	GetMetricHistories(
 		ctx context.Context,
-		experimentIDs []string, runIDs []string, metricKeys []string, viewType request.ViewType, limit int32,
+		namespaceID uint,
```

Comment:
Don't we want to test these repositories?

Reply:
no, we usually don't need to test `repositories`. `integration` tests do this for us.